src/dispatching/dialogue/storage/sqlite_storage.rs

```diff
@@ -115,20 +119,22 @@ where
     }
 }
 
-#[derive(sqlx::FromRow)]
-struct DialogueDbRow {
-    dialogue: Vec<u8>,
-}
-
 async fn get_dialogue(
     pool: &SqlitePool,
     chat_id: i64,
 ) -> Result<Option<Box<Vec<u8>>>, sqlx::Error> {
```

Comment:
What's the point in `Box`ing `Vec` anyway?

Reply:
Fixed.